docs/guides/software-lifecycle/security/security-scanning/README.md

```diff
@@ -0,0 +1,208 @@
+# Security Scanning
+
+<pre align="center">Guide to code scanning for security issues.</pre>
+
+## Introduction
+
+**Background**: Software security is critical in modern systems with application code at its root. Identifying and addressing vulnerabilities rapidly mitigates risk and limits the potential surface area of attacks. We recommend [NASA's SCRUB platform](https://github.com/nasa/scrub) to manage code scanning by identifying, orchestrating and aggregating security information. SCRUB's GitHub implementation wraps [CodeQL](https://codeql.github.com/) results into compact, curated reports that highlight security assessments and are suitable for ingestion by automated reporting tools. A small configuration is appended to an existing CodeQL configuration (`codeql-config.yml` file) that specifies security analyses and reporting properties.
+
+**Use Cases**:
+- Standardized security reports that enables rapid interchange of scanning tools.
```

Comment:
Since this is an enumeration and each isn't a complete sentence, no need for periods here.

Reply:
Done.